angular/core/components/inbox_page/inbox_page.haml

```diff
@@ -4,8 +4,13 @@
       %h1.lmo-h1-medium.inbox-page__heading{translate: "inbox_page.unread_threads"}
       %loading{ng-if: "inboxPage.loading()"}
       .inbox-page__threads{ng-if: "!inboxPage.loading()"}
-        .inbox-page__no-threads{ng-show: "!inboxPage.hasThreads()"}
+        .inbox-page__no-threads{ng-show: "!inboxPage.hasThreads() && !inboxPage.noGroups()"}
```

Comment:
As a minor nitpick, it's confusing to parse 'hasThreads' and 'noGroups' together, so this would read better as `inboxPage.hasGroups()`